docs/GETTING_STARTED.md

```diff
@@ -113,15 +122,15 @@ model = init_detector(config_file, checkpoint_file, device='cuda:0')
 img = 'test.jpg'  # or img = mmcv.imread(img), which will only load it once
 result = inference_detector(model, img)
 # visualize the results in a new window
-show_result(img, result, model.CLASSES)
+model.module.show_result(img, result)
```

Comment:
`model.module` is unnecessary because model is not a DataParallel or DistributedDataParallel.